test/modifiers/flatten.test.js

```diff
@@ -0,0 +1,155 @@
+var expect = require('chai').expect;
+var _ = require('lodash');
+
+var runBasicStreamTests = require('../_utilities/runBasicStreamTests.js');
+var getReadableStream = require('../_utilities/getReadableStream.js');
+
+var flatten = require('../../').flatten;
+var batch = require('../../').batch;
+
+function buff(str) {
+  return new Buffer(str);
+}
+
+describe('[flatten]', function () {
+  (function () {
+    var data = ['item1', 'item2', 'item3'];
+    var expected = data.map(buff);
+    var objData = [{ a: 1}, [1, 2, 3], true, 'string', 42, _.noop];
+    var objExpected = objData.reduce(function (memo, val) {
```

Comment:
Can this be spelled out more explicitly so it's easier to follow? It seems like the expectation should be `[{ a: 1}, 1, 2, 3, true, 'string', 42, _.noop]` which is easier to understand (I think).